app/models/user.rb

```diff
@@ -375,6 +375,15 @@ def identity
     @identity ||= UserIdentity.find(uuid)
   end
 
+  def onboarding
+    if Flipper.enabled?(:veteran_onboarding_beta_flow, self) && user_account_uuid.present?
+      @onboarding ||= VeteranOnboarding.find_or_create_by(user_account_uuid:)
```

Comment:
you should be able to just pass in the `user_account`, makes things a big cleaner and you can let `ApplicationRecord` take care of some of the querying

Reply:
👍

Reply:
Hmm that actually didn't work.  Could be that when I created the `VeteranOnboarding` model, I just made a column called `user_account_uuid`